cmd/sebak/cmd/run.go

```diff
@@ -360,6 +364,26 @@ func parseFlagsNode() {
 		threshold = int(tmpUint64)
 	}
 
+	// tx pool limits (client,node)
+	{
+		limits := strings.Split(flagTxPoolLimit, ",")
+	L:
+		for i, l := range limits {
+			switch i {
+			case 0:
+				if txPoolClientLimit, err = strconv.ParseUint(l, 10, 64); err != nil {
+					cmdcommon.PrintFlagsError(nodeCmd, "--txpool-limit", err)
+				}
+			case 1:
+				if txPoolNodeLimit, err = strconv.ParseUint(l, 10, 64); err != nil {
+					cmdcommon.PrintFlagsError(nodeCmd, "--txpool-limit", err)
+				}
+			default:
+				break L
```

Comment:
I think we should issue error if the user provides more than 2 args

Reply:
@Geod24 What kinds of issue?

Reply:
"issue" as a verb means "to give" or "to make".
So: "I think we should really have an error instead of the `default`"

Reply:
Oh, I got it.

Reply:
`error: invalid '--txpool-limit'; wrong format: format:<client-limit>[,<node-limit]` 
How about it? @Geod24

Reply:
Sounds good